librz/util/path.c

```diff
@@ -180,3 +199,33 @@ RZ_API RZ_OWN char *rz_path_home_expand(RZ_NULLABLE const char *path) {
 
 	return rz_path_home(path + 1);
 }
+
+/**
+ * \brief Return a canonicalized absolute path. Expands all symbolic links and resolves
+ * references to /./, /../ and extra '/' characters.
+ *
+ * \param path Original file path.
+ * \return New canonicalized absolute path.
+ */
+RZ_API RZ_OWN char *rz_path_realpath(RZ_NULLABLE const char *path) {
+	if (!path) {
+		return NULL;
+	}
+#if __UNIX__
```

Comment:
Instead of this i would use HAVE_REALPATH computed from meson

Reply:
Also went ahead and added it in file.c